src/Integration.UnitTests/Binding/BindingWorkflowTests.cs

```diff
@@ -362,6 +444,43 @@ private void ConfigureProfileExport(BindingWorkflow testSubject, RoslynExportPro
             testSubject.LanguageToGroupMapping[language] = group;
         }
 
+        private static void VerifyNuGetPackgesDownloaded(IEnumerable<PackageName> expectedPackages, BindingWorkflow testSubject)
+        {
+            var expected = expectedPackages.ToArray();
+            var actual = testSubject.NuGetPackages.Select(x => new PackageName(x.Id, new SemanticVersion(x.Version))).ToArray();
+
+            Assert.AreEqual(expected.Length, actual.Length, "Different number of packages.");
+
+            for (int i = 0; i < expected.Length; i++)
+            {
+                Assert.IsTrue(expected[i].Equals(actual[i]), $"Packages are different at index {i}.");
+            }
+        }
+
+        private static void VerifyAdditionalFilesDownloaded(IEnumerable<AdditionalFile> expectedFiles, BindingWorkflow testSubject)
+        {
+            var expected = expectedFiles.ToArray();
+            var actual = testSubject.AdditionalFiles.ToArray();
+
+            Assert.AreEqual(expected.Length, actual.Length, "Different number of files.");
+
+            for (int i = 0; i < expected.Length; i++)
+            {
+                Assert.AreEqual(expected[i].FileName, actual[i].FileName, $"Files have different names at index {i}.");
+                CollectionAssert.AreEqual(expected[i].Content, actual[i].Content, $"Files have different content at index {i}.");
+            }
+        }
+
+        private static void VerifyAdditionalFileAddedToProject(Project vsProject, MSBuild.Project msbuildProject, string additionalFilePath)
+        {
+            string relativePath = PathHelper.CalculateRelativePath(vsProject.FullName, additionalFilePath);
+
+            var additionalFileItems = msbuildProject.Items.Where(x => x.ItemType == Constants.AdditionalFilePropertyKey);
+
+            Assert.IsTrue(additionalFileItems.Select(x => x.UnevaluatedInclude).Contains(relativePath), "AdditionalFile is missing");
```

Comment:
Probably worth including in the assert message the relative file that was missing